### PR TITLE
sql: improve privileges docs

### PR DESCRIPTION
**Pull request by morgo**
I noticed a few small errors in the privileges page.  While few users use column-level privileges, I think it is still worth calling out on the compatibility page.

### CODE REVIEW

**Comment by morgo** [comment]
PTAL @tiancaiamao

File: sql/privilege.md [hunk 1]
```diff
@@ -289,7 +294,7 @@ When TiDB starts, some privilege-check tables are loaded into memory, and then t
 If an immediate effect is needed when you modify the `grant` table, you can run the following command:
 
 ```sql
-flush privileges
+flush privileges;
```
Comment (kennytm) [inline sql/privilege.md]: Perhaps use ALL CAPS here

Comment (morgo) [reply sql/privilege.md]: @lilin90 do we have a style for SQL commands?  I like to write keywords in caps as kenny suggests, but many of our existing examples are lower case.

Comment (kennytm) [reply sql/privilege.md]: If the style is all-lowercase then the `DROP USER 'test';` statement above should be changed to `drop user 'test';`.

Comment (morgo) [reply sql/privilege.md]: @lilin90 There are quite a few inconsistencies: `CREATE USER`, `REVOKE` are in caps.  I can go through and change once I have confirmation of the style.

Comment (lilin90) [reply sql/privilege.md]: @morgo Thanks! Currently, we don't have a strict SQL style. But to improve the consistency, we can now establish one: "Capitalize all SQL keywords and reserved words." Later, I'll update that in docs-cn and tell other members to follow this style.

Comment (morgo) [reply sql/privilege.md]: @lilin90 thanks, I've updated all keywords to caps.  PTAL

**Comment by tiancaiamao** [comment]
LGTM

**Pull request merged**